src/OrchardCore.Modules/OrchardCore.Features/Views/Admin/Features.cshtml

```diff
@@ -2,36 +2,31 @@
 @using OrchardCore.Modules;
 @model FeaturesViewModel
 
-@{
-}
 <div class="row">
     <div class="form-group col mb-n1">
         <h1>@RenderTitleSegments(T["Features"])</h1>
     </div>
-    <div class="form-group col mb-n1">
-        <div class="dropdown float-right order-md-1">
-            <button class="btn btn-sm btn-secondary dropdown-toggle" type="button" id="bulk-action-menu-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                @T["Bulk Actions"]
-            </button>
-            <div class="dropdown-menu dropdown-menu-right" aria-labelledby="bulk-action-menu-button">
-                <a class="dropdown-item" href="#" data-action="@nameof(FeaturesBulkAction.Disable)">@T["Disable"]</a>
-                <a class="dropdown-item" href="#" data-action="@nameof(FeaturesBulkAction.Enable)">@T["Enable"]</a>
-                <a class="dropdown-item" href="#" data-action="@nameof(FeaturesBulkAction.Toggle)">@T["Toggle"]</a>
-            </div>
-        </div>
-    </div>
 </div>
-
-<form autocomplete="off">
+<form autocomplete="off" class="sticky">
```

Comment:
Can you try with `position-sticky` instead?

Reply:
I didn't know that bootstrap tried to simulate all the things ;)